### DISCUSSION TITLE
Keeping two copies in the merged FS

### DISCUSSION THREAD

**Discussion by ubrfkr** [trapexit/mergerfs#961] [Q&A] [answered]
Does anyone know how I could create some redundancy by having mergerfs maintain a copy of files in a given directory on multiple disks in the array?

**Comment by trapexit** [comment, marked as answer]
As mentioned in the docs if you don't see something mentioned in the docs then it doesn't exist in the product. 

That said something like can be accomplished out of band: https://github.com/trapexit/mergerfs-tools#mergerfsdup

**Reply by ubrfkr** [reply]
Thank you.
